data-service/package.json

```diff
@@ -16,10 +16,8 @@
     "@babel/preset-env": "^7.10.4"
   },
   "scripts": {
-    "build": "babel graphql/src -d graphql/lib",
-    "start": "node graphql/lib/server.js",
+    "build": "babel server/src -d server/lib",
+    "start": "node server/lib/server.js",
     "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "mSupply Foundation",
```

Comment:
Hm (this is just a question), is this something that has a convention yet, or pending discussion on a convention? 
I took a quick look through the other projects and noticed quite a bit of a variety. E.g. mobile uses
 "Sustainable Solutions (NZ) Ltd", and V1 of UC has Chris listed as the author 😆 .

Reply:
No convention yet ( clearly! ) 
The direction I would expect is for everything to be under `The mSupply Foundation` or the `mSupply Foundation`? So yes, still a little discussion required